index.html

```diff
@@ -71,6 +71,24 @@
           <a href="#" class="btn btn-white">see the menu</a>
         </div>
       </div>
+      <div class="about">
```

Comment:
There's a good semantic tag name for such case, check docs at mdn to know more
```suggestion
      <section class="about">
```